### PR TITLE
Bdd read payments

### PR DESCRIPTION
**Pull request by nvme5**
Added feature to read payments - existing and non existing
modified the code in payments.py to show the proper error message (workaround for now)

### CODE REVIEW

**Comment by nvme5** [comment]
Let me check that

**Comment by nvme5** [comment]
Updated the failing test case, all should be good now

File: tests/features/payments.feature [hunk 1]
```diff
@@ -5,7 +5,7 @@ Feature: The payments API
 
 Background:
     Given the following "payments"
-        | nickname   | user_id | payment_type | user_name    | expires | card_type  | card_number      | user_email | is_linked | 
+        | nickname   | user_id | payment_type | user_name    | expires | card_type  | card_number      | user_email | is_linked |
```
Comment (jdgumz) [inline tests/features/payments.feature]: Lol, nice - you caught a rogue space character!

**Comment by odoric27** [comment]
@nvme5 could you fill out the AC for this story before closing it out? thanks!

File: tests/features/payments.feature (hunk 1 above)
Comment (nvme5) [inline tests/features/payments.feature]: Lol, I guess my editor caught it

**Pull request merged**